feat(curriculum): Add fCC Authors Page Project to JS Cert (#49826)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/641daa5ea050f24a7cade6e6.md

---
id: 641daa5ea050f24a7cade6e6
title: Step 26
challengeType: 0
dashedName: step-26
---

# --description--

> Now, disable the button and set the text content of the button to "No more data to load". You can disable it by setting its `disabled` property to `true`.

Reviewer: ```suggestion
If this condition is met, disable the button by setting its `disabled` property to `true`. Also, set the text content of the button to `No more data to load`.
```
